packages/webapp/src/components/Form/InputBase/InputBaseLabel/index.tsx

```diff
@@ -26,14 +26,15 @@ export type InputBaseLabelProps = {
   hasLeaf?: boolean;
   toolTipContent?: string;
   icon?: ReactNode;
+  labelStyles?: React.CSSProperties;
 };
 
 export default function InputBaseLabel(props: InputBaseLabelProps) {
   const { t } = useTranslation();
 
   return (
     <div className={styles.labelContainer}>
-      <Label>
+      <Label style={props.labelStyles}>
```

Comment:
I am wondering if it should be possible to alter baseLabelStyles? I think we want all to be consistent or to have an option for primary or secondary or something along those lines. 

So you could probably add those styles here directly if they are what we want on all from now on.

Reply:
Can you remind us to discuss this with Loïc when he gets back? I love the new styles and I hope we do use them throughout app, but unfortunately we didn't think to have that discussion before Loïc left, so I don't feel quite ready to push that thorough a change now.

Separately from that, I do like having the ability to style the label differently if needed, so I would like to keep (restore?) the label classes behaviour. But you're right that it should be for true one-offs and not for a systemic change like this.